src/main/java/de/hbz/lobid/helper/JsonConverter.java

```diff
@@ -320,11 +297,7 @@ private void createObject(String uri, Map<String, Object> newObject) {
 				}
 			} else {
 				if (!mainSubjectOfTheResource.equals(s.getObject().stringValue())) {
-					if (!statementVisited(s)) {
-						createObject(newObject, s, e);
-					} else {
-						createLeafObject(newObject, s, e);
-					}
+					createObject(newObject, s, e);
 				} else {
```

Comment:
@jschnasse please have a look. The solution for me was to simply remove the whole `createLeafObject`-method. Test are all fine. What was the original idea of the method?